data/map planets.txt

```diff
@@ -2877,7 +2877,7 @@ planet Muspel
 		fleet "Small Deep Security" 14
 
 planet Mutiny
-	attributes pirate
+	attributes pirate core
```

Comment:
This should also have "core pirate", and the attributes should be listed alphabetically.